app/admin/dashboard.rb

```diff
@@ -21,6 +21,10 @@ def print_stats(relation)
         panel I18n.t('activerecord.models.user.other') do
           print_stats(User)
         end
+
+        panel I18n.t('active_admin.charts.sign_ups_per_month') do
+          render 'sign_ups_per_month_partial', { users: User.group_by_month(:created_at).count }
```

Comment:
this shouldn't be called `<name>_partial`, just `<name>`, right? the `_` prefix should be enough